xbmc/cores/VideoPlayer/VideoRenderers/HwDecRender/RendererVTBGL.cpp

```diff
@@ -73,7 +73,7 @@ EShaderFormat CRendererVTB::GetShaderFormat()
 bool CRendererVTB::LoadShadersHook()
 {
   CLog::Log(LOGNOTICE, "GL: Using CVBREF render method");
-  m_textureTarget = GL_TEXTURE_RECTANGLE_ARB;
+  m_textureTarget = GL_TEXTURE_RECTANGLE;
```

Comment:
now that GL_TEXTURE_RECTANGLE is gone from the other parts, I would check if this is still ok here. I don't think that sw and vtb decoding on osx need different values.

Reply:
@FernetMenta if I remove `GL_TEXTURE_RECTANGLE` in VTB hw rendering is broken on macos.
So I assume it's still needed.

Reply:
I don't think it is really needed. More likely that some other code has hardcoded this assumption. I can't think of any reason why macos should not support TEXTURE_2D

Reply:
I have no macos at hand, so I can't go any further with this.
If we can remove this in VTB then I could drop a lot more code and also remove the special handling in all the shaders.
Any help is highly appreciated.

Reply:
Forgot to mention:
Broken = green screen

Reply:
confused :)
if current PR is ok on all platforms, even with GL_TEXTURE_RECTANGLE, go ahead. I just wanted to ask if this is the case.

Reply:
It needs more testing, I can only test on linux atm.
But this is the last occurance of GL_TEXTURE_RECTANGLE and it's removal would break macos hw rendering.
So it would be nice if we could get rid of it.
Anyway I'll wait for v19 alpha.